packages/postgresql/configuration-schema.json

```diff
@@ -6,16 +6,8 @@
       "type": "string",
       "description": "Postgres instance host URL or IP address",
       "minLength": 1,
-      "anyOf": [
-        {
-          "format": "uri"
-        },
-        {
-          "format": "ipv4"
-        }
-      ],
       "examples": [
-        "https://some-host.compute-1.amazonaws.com",
+        "some-host.compute-1.amazonaws.com",
```

Comment:
Is removing https intentional? https would still work right?

We're trying to relax the validation, not change the behaviour or documentation

Reply:
For database host url you need to use `hostname` or `ipAddress` only
There no `http` or `https` protocol